### PR TITLE
html-web-page exercise - Brandon Chin

### PR DESCRIPTION
**Pull request by brandonechin**
2022-12-06 14_24_14-HTML Web Page

### CODE REVIEW

**Comment by brandonechin** [comment]
2022-12-06 14_28_10-html-web-page

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+  <p>Like what you see? Become a part of the JOTMC family today!</p>
+  <form>
+    <label for="name"></label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:70]: There is no text labeling this form control element, so no `label` element is necessary

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    <input type="text" name="name" placeholder="Your Name" id="name">
+    <label for="SelectPlan"></label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:72]: There is no text labeling this form control element, so no `label` element is necessary

File: html-web-page/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    <select name="plan" id="SelectPlan">
+      <option>Select Your Plan:</option>
+      <option value="Basic Plan">Basic Plan</option>
+      <option value="Deluxe Plan">Deluxe Plan</option>
+      <option value="Jelly Giant Plan">Jelly Giant Plan</option>
+    </select>
+    <label for="submit"></label>
```
Comment (codingcodymiller) [inline html-web-page/index.html:79]: There is no text labeling this form control element, so no `label` element is necessary

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by brandonechin** [comment]
2022-12-06 15_03_10-index html - c1222-code-solutions  Dev Container  - Visual Studio Code

**Pull request merged**